packages/monaco-editor-wrapper/src/wrapper.ts

```diff
@@ -33,11 +33,27 @@ export type EditorConfig = {
     diffEditorOptions?: editor.IStandaloneDiffEditorConstructionOptions;
 }
 
+export type WebSocketStartOptions = {
```

Comment:
What about creating only one new type like (I left out the comments to make it shorter):
```typescript
export type WebSocketCallOptions = {
    onCall: () => void;
    reportStatus?: boolean;
}
```
You can use the type then for both `webSocketStartOptions` and `webSocketStopOptions`.

Reply:
Would you prefer the comments to be left out? Personal preference is for them to be there - allows the IDE to describe what type is used for instead of sifting through the code :)

Reply:
Please leave the comments in. I just omitted them above to make my comment smaller. 🙂 

Just looked at your update. I had something like this in mind. WDYT?
```typescript
export type WebSocketCallOptions  = {
    /** Adds handle on call of fundction */
    onCall: () => void;
    /** Reports Status Of Language Client */
    reportStatus?: boolean;
}

export type WebSocketConfigOptions = {
    secured: boolean;
    host: string;
    port: number;
    path: string;
    startOptions?: WebSocketCallOptions;
    stopOptions?: WebSocketCallOptions;
}